code-challenges.js

```diff
@@ -4,38 +4,143 @@
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+var fib = [0, 1];
 
+// i want to loop throught the index in the array
+       for (var i=fib.length; i < 11; i++) {
 
-// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+// i put the formula in
+        fib[i] = fib[i-2] + fib[i-1];
+    }
 
-var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+// shaaabang and console log that lil fibonacci!!!!!!!!
+    console.log(fib);
+
+// --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest
+
+
+
+// THIS WAS AWSOME!!! I GO THE NUMBERS IN THROOUGH THE FUNCTION. 
+// BUT UNFORTUNALTRLY I CAN T SORT IT. I TAKE IT BACK!!!!Q!! I CAN SORT!!!!!!!!!!!!!!
+// BECAUSE I HAVE SPENT LIKE 10 HOURS ON THIS PROBLEM AND THE SHOW MUST GO ON
+
+
+
+var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"];
 // Expected output: [-9, 9, 7, 199]
-var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+
+fullArr1.sort();
+
+function filterArr(fullArr1) {
+    var onlyNums = [];
+    for (var i = 0; i < fullArr1.length; i++) {
+        if (typeof fullArr1[i] === "number" && fullArr1[i] % 2 === 1 | fullArr1[i] % 2 === -1) {
+            onlyNums.push(fullArr1[i]);
+        }
+    }
+     
+    return onlyNums;
+}
+console.log(filterArr(fullArr1));
```

Comment:
good job getting all the wya to the sorting. this one was challenging. did the assessment review help with this?